### ISSUE TITLE
[Paper-Cut:UI] Poor usability in AML notebooks:  Folders and files management in AML notebooks UI is not intuitive when creating folders or files 

### ISSUE THREAD

**Issue by CESARDELATORRE** [danielsc/azureml-workshop-2019#23]
In AML notebooks UI, when creating a folder or file, it should be created into the original folder that was previously selected before clicking to "New Folder".
Instead, in the dialog window, the user needs to select "again" the parent folder where the child folder of file is going to be created.

This causes poor usability and in most cases you create the child folder or file in a folder you didn't want and need to re-create it again...

**Comment by danielsc** [comment]
agree: https://msdata.visualstudio.com/Vienna/_workitems/edit/583746
